examples/example2.md

```diff
@@ -1,68 +1,72 @@
 ## Example 2
 
-> **Question:** Get the pressure measures of patients born after 1970, together with their gender and birthdate
+> **Question:** Get clinical information about patients that had a general examination because of Coronavirus
+>
+> Here we get diagnosis codes for finished encounters. We get the period of the encounter and some information about the patient such as his age, gender and if he is alive.
 
-Note that http://loinc.org|55284-4 corresponds to blood pressure.
+Note that we use the following SNOMED CT codes:
+- 162673000: General examination of patient (procedure)
+- 840546002: Exposure to severe acute respiratory syndrome coronavirus 2 (event)
+- _(840539006: Disease caused by severe acute respiratory syndrome coronavirus 2 (disorder))_
```

Comment:
we're not using this code 840539006, can we rm it?

Reply:
Yes indeed, thank you !